projects/packages/my-jetpack/src/class-rest-product-data.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Sets up the Product Data REST API endpoints.
+ *
+ * @package automattic/my-jetpack
+ */
+
+namespace Automattic\Jetpack\My_Jetpack;
+
+use Automattic\Jetpack\Connection\Client;
+use WP_Error;
+
+/**
+ * Registers the REST routes for Product Data
+ */
+class REST_Product_Data {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		register_rest_route(
+			'my-jetpack/v1',
+			'site/product-data',
+			array(
+				'methods'             => \WP_REST_Server::READABLE,
+				'callback'            => __CLASS__ . '::get_all_product_data',
+				'permission_callback' => __CLASS__ . '::permissions_callback',
+			)
+		);
+	}
+
+	/**
+	 * Checks if the user has the correct permissions
+	 */
+	public static function permissions_callback() {
+		return current_user_can( 'manage_options' );
+	}
+
+	/**
+	 * Gets the product data for all products
+	 *
+	 * @return array|WP_Error
+	 */
+	public static function get_all_product_data() {
+		$site_id        = \Jetpack_Options::get_option( 'id' );
+		$wpcom_endpoint = sprintf( 'sites/%d/jetpack-product-data/backup?locale=%2$s&force=wpcom', $site_id, get_user_locale() );
```

Comment:
This looks like the backup specific endpoint, is that intentional?

Reply:
oops, no it is not, let me remove that!

Doesn't matter too much right now since it's not really being used but yes that was accidental